lib/mongo/server/connection_pool/available_stack.rb

```diff
@@ -283,15 +271,15 @@ def max_idle_time
         #   if the option is set.
         #
         # @example Close the stale sockets
-        #   queue.close_stale_sockets!
+        #   stack.close_stale_sockets!
         #
         # @since 2.5.0
         def close_stale_sockets!
```

Comment:
Was this method audited for correct operation given the other changes in the diff? I see that it calls `connection.disconnect!` instead of `close_connection` for example.